### PR TITLE
[log-shipper] Elasticsearch datastream support

### PR DESCRIPTION
**Pull request by yalosev**
## Description
Storing data in datastreams is a more flexible way for elasticsearch users. Data streams are well-suited for logs.

## Why we need it and what problem does it solve?
Datastreams are more appropriate for storing logs and metrics.
Also add support for Elasticsearch <= 6.X

## Changelog entries

```changes
module: log-shipper
type: feature
description: Support storing data in Elasticsearch datastreams
```

### CODE REVIEW

File: modules/460-log-shipper/crds/doc-ru-cluster-log-destination.yaml
Comment (nabokihms) [inline modules/460-log-shipper/crds/doc-ru-cluster-log-destination.yaml:77]: Hmm, it looks like there is an error with the indent.

**Review by nabokihms — CHANGES_REQUESTED** [review]
Overall, it looks good. Could you please squash commits and rebase the branch onto the main to run workflows.

File: modules/460-log-shipper/crds/doc-ru-cluster-log-destination.yaml
Comment (yalosev) [reply modules/460-log-shipper/crds/doc-ru-cluster-log-destination.yaml]: fixed

File: modules/460-log-shipper/docs/USAGE.md
Comment (nabokihms) [inline modules/460-log-shipper/docs/USAGE.md:181]: It would help if you used the double `#` sign here, and the same goes for the Russian doc.
```suggestion
## Elasticsearch <= 6.X usage
```

image

File: modules/460-log-shipper/docs/USAGE_RU.md
Comment (nabokihms) [inline modules/460-log-shipper/docs/USAGE_RU.md:181]:
```suggestion
## Поддержка elasticsearch <= 6.X
```

**Review by nabokihms — CHANGES_REQUESTED** [review]

**Review by nabokihms — APPROVED** [review]

**Comment by nabokihms** [comment]
Let's wait for @konstantin-axenov for the final approval.

**Pull request merged**
